src/server/src/main/java/com/spotify/reaper/cassandra/JmxProxy.java

```diff
@@ -287,14 +289,14 @@ public String getDataCenter(String host) {
     checkNotNull(ssProxy, "Looks like the proxy is not connected");
 
     return Lists.transform(
-            Lists.newArrayList(((StorageServiceMBean) ssProxy).getTokenToEndpointMap().keySet()), s -> new BigInteger(s));
+        Lists.newArrayList(((StorageServiceMBean) ssProxy).getTokenToEndpointMap().keySet()), s -> new BigInteger(s));
   }
 
   public Map<List<String>, List<String>> getRangeToEndpointMap(String keyspace) throws ReaperException {
     checkNotNull(ssProxy, "Looks like the proxy is not connected");
     try {
       return ((StorageServiceMBean) ssProxy).getRangeToEndpointMap(keyspace);
-    } catch (Exception e) {
+    } catch (RuntimeException e) {
```

Comment:
This could leave us with uncatched exceptions, which sucks. Why are you changing this ?

Reply:
No that's not possible.
Here the only other exceptions, under `Exception`, would be checked exceptions. The only checked exception that could slip through here is if `getRangeToEndpointMap(keyspace)` threw a ReaperException, otherwise the code would not compile.  `getRangeToEndpointMap(keyspace)`  does not throw any checked exceptions.

The code won't catch Errors (or Throwables), but it didn't before either.